README.md

```diff
@@ -104,7 +106,7 @@ we talk to clients.__
 
    ![Formatting](https://i.imgur.com/cqZZ8HC.gif)
 
- - Renaming via HaRe
+ - Renaming via HaRe (NOTE: HaRe it's not available temporary)
```

Comment:
Englisho: "HaRe is temporarily disabled"

Reply:
Corrected with https://github.com/mpickering/haskell-ide-engine/pull/93